lib/stale.js

```diff
@@ -198,8 +198,27 @@ module.exports = class Stale {
   }
 
   since (days) {
-    const ttl = days * 24 * 60 * 60 * 1000
-    let date = new Date(new Date() - ttl)
+    const daysInMilliseconds = 24 * 60 * 60 * 1000
+    const now = Date.now()
+    let daysToCount = 0
```

Comment:
Maybe moving this inside the `if` and renaming to `daysAgoSkippingWeekends` could clarify I bit? I was looking for it to be used again later, but it's not.